File: Java言語/基礎知識用の勉強/3条件分岐勉強/条件分岐.java
class 条件分岐 {
	public static void main(String[] args){
		System.out.print("hello World.\n");

		int asakuno = 20210919;
		int tomohiro = 20210920;

		if ( asakuno+1 == tomohiro ) {
			System.out.print( "20210920 == 20210920：" );
			System.out.println( asakuno+1 == tomohiro );	// 20210920 == 20210920：true
		}

		if ( asakuno == tomohiro ) {
			System.out.print( "20210919 == 20210920：" );
			System.out.println( asakuno == tomohiro );	// 
		}
		else {
			System.out.print( "20210919 != 20210920：" );
			System.out.println( asakuno != tomohiro );	// 20210919 != 20210920：true
		}

		if ( asakuno == tomohiro ) {
			System.out.print( "20210919 == 20210920：" );
			System.out.println( asakuno == tomohiro );	//
		}
		else if ( asakuno+1 != tomohiro ) {
			System.out.print( "20210920 != 20210920：" );
			System.out.println( asakuno+1 != tomohiro );	//
		}
		else {
			System.out.print( "20210919 < 20210920：" );
			System.out.println( asakuno < tomohiro );	// 20210919 < 20210920：true
		}

	}
}

// vim: set ts=4 sts=4 sw=4 tw=0 ff=unix fenc=utf-8 ft=java noexpandtab:

File: Go言語/基礎知識用の勉強/3条件分岐勉強/実行済み/条件分岐.go
package main

import "fmt"

func main() {
	var asakuno = "asakuno"

	if asakuno == "朝来野智博" {
		fmt.Println("朝来野智博")
	} else if asakuno == "asakuno" {
		fmt.Println("asakuno")
	} else{
		fmt.Println("その他")
	}
	// 出力結果：asakuno
	fmt.Println("-----------------------")
}

/* vim: set ts=4 sts=4 sw=4 tw=0 ff=unix fenc=utf-8 ft=go noexpandtab: */
